note_taking-explainer.md

```diff
@@ -71,15 +114,8 @@ changing the semantics of the member.
 
 ### A General Capability Member
 
-There are existing general capabilities approaches for other functionality of
-web apps/pages:
-
-* [Web Share Target](https://w3c.github.io/web-share-target/)
-* [File Handling](https://github.com/WICG/file-handling/blob/main/explainer.md)
-* [Protocol Handlers](https://html.spec.whatwg.org/multipage/system-state.html#dom-navigator-registerprotocolhandler)
-* [Declarative Web Actions](https://github.com/slightlyoff/declarative_web_actions)
-
-So perhaps we could do something similar with a more general-purpose field like:
+Perhaps we should use a more general-purpose field, where each separate
```

Comment:
Rephrase this. "Perhaps we should" suggests you're considering it, but really this is something we've considered and discarded.

Reply:
Done.